databases/mariadb-10.10/Portfile

```diff
@@ -1,287 +1,19 @@
-will be lost during port upgrades, deactivations or activations so you
-are advised to not make changes here. Currently
-${prefix}/etc/${name_mysql}/macports-default.cnf contains only one
-directive; to disable networking. With disabled networking it is
-possible to install and have running all the MacPorts mysql ports
-simultaneously.
-"
-}
-
-subport ${name_mysql}-server {
-
-    revision            ${revision_server}
+subport mariadb-10.10-server {
+    replaced_by         mariadb-10.11-server
+    revision            1
     license             BSD
-    description         Run ${name_mysql} as server
-    long_description    {*}${description}
-
-    supported_archs     noarch
-    distfiles
-
-    depends_run         port:${name_mysql}
-
-    if {"darwin" eq ${os.platform} && ${os.major} > 8} {
-        set mysqluser       _mysql
-    } else {
-        set mysqluser       mysql
-    }
-    add_users ${mysqluser} group=${mysqluser} realname=MySQL\ Server
-
-    pre-extract {
-        copy ${filespath}/org.macports.mysql-server.plist ${workpath}/org.macports.${subport}.plist
-    }
-
-    post-patch {
-        reinplace "s|@NAMEMYSQL@|${name_mysql}|g" \
-            ${workpath}/org.macports.${subport}.plist
-        reinplace "s|@PREFIX@|${prefix}|g" \
-            ${workpath}/org.macports.${subport}.plist
-        reinplace "s|@SUBPORT@|${subport}|g" \
-            ${workpath}/org.macports.${subport}.plist
-        reinplace "s|@USER@|${mysqluser}|g" \
-            ${workpath}/org.macports.${subport}.plist
-        reinplace "s|@GROUP@|${mysqluser}|g" \
-            ${workpath}/org.macports.${subport}.plist
-    }
-
-    use_configure       no
-
-    build {}
-
-    destroot {
-        xinstall -d -m 755 ${destroot}${prefix}/etc/LaunchDaemons/org.macports.${subport}
-        xinstall -m 0644 -o root -W ${workpath} \
-            org.macports.${subport}.plist \
-            ${destroot}${prefix}/etc/LaunchDaemons/org.macports.${subport}
-        xinstall -d -m 755 ${destroot}/Library/LaunchDaemons
-        ln -s ${prefix}/etc/LaunchDaemons/org.macports.${subport}/org.macports.${subport}.plist \
-            ${destroot}/Library/LaunchDaemons/org.macports.${subport}.plist
-        xinstall -m 755 -o root -d \
-            ${destroot}${prefix}/var/run
-        xinstall -m 755 -o ${mysqluser} -g ${mysqluser} -d \
-            ${destroot}${prefix}/etc/${name_mysql} \
-            ${destroot}${prefix}/var/db/${name_mysql} \
-            ${destroot}${prefix}/var/log/${name_mysql} \
-            ${destroot}${prefix}/var/run/${name_mysql}
-        destroot.keepdirs-append  \
-            ${destroot}${prefix}/var/db/${name_mysql} \
-            ${destroot}${prefix}/var/log/${name_mysql} \
-            ${destroot}${prefix}/var/run/${name_mysql}
-    }
-
-    notes "
-If this is a new install you might want to run:
-
-\$ sudo -u ${mysqluser} ${prefix}/lib/${name_mysql}/bin/mysql_install_db
-"
 }
 
-livecheck.type  none
+# Remove after 2028-04-02
```

Comment:
Is there a reason this needs to be kept for 3 years instead of the usual 1?

Reply:
nope - tried guessing what it should be base upon incomplete / misunderstood information